src/Public/js/sis.selector.js

```diff
@@ -23,15 +25,25 @@ var Selector = function () {
       }
 
       if (locked) {
-        myradio.createDialog("Error", "Could not change studio.<br>Studio selector is currently locked out.");
+        myradio.createDialog("Selector Error", "Could not change studio.<br>Studio Selector is currently locked out.");
         return;
       }
+
+      if (confirm == studio) {
+        selStudio(studio); // If they click the same source twice, select it.
```

Comment:
should reset confirm to 0? Should probably do so in the case that confirm != 0, as well. Maybe outside the if else?

Reply:
Ah, woops. My idea was that if you selected the wrong one, you didn't have to cancel it by clicking another source, (confirm=0), but it would just ask you to confirm the second source instead. But yes, confirm=0 should happen when you confirm the selection, since you could potentially get a case where you didn't confirm a quick outside-initiated change.